docs/content/policy-reference.md

```diff
@@ -1024,6 +1026,26 @@ example_verify_resource {
 }
 ```
 
+##### Unsigned Payload Request Signing Example
+The [AWS S3 request signing API](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html)
+supports unsigned payload signing option. This example below shows s3 request signing with payload signing disabled. 
+
+```live:providers/aws/sign_req_basic:module
```

Comment:
You'll need to rename this to get the website to build.  `sign_req_basic:module` -> `sign_req_unsign:module` for instance.

Reply:
Done.